app-foundation/README.md

```diff
@@ -0,0 +1,57 @@
+# terraform-example-foundation-app/app-foundation
+
+## Overview
+The `app-foundation` folder contains several distinct Terraform configurations that can supplement the CFT to deploy the necessary foundation for the Bank of Anthos example application. Each folder contains its own directory that must be applied separately, and in sequence, to correctly deploy the application. The necessary steps are outlined below for the relevant folders.
+
+### [3. networks](./3-networks/)
+
+This step focuses on creating a Shared VPC per environment (`development`, `non-production` & `production`) that is consistent with the Bank of Anthos application's configuration and has a reasonable security baseline. Currently this includes:
+
+- Example subnets for `development`, `non-production` & `production` inclusive of secondary ranges for the Bank of Anthos application.
+- Additional firewall rules that are necessary for the configuration to work. **Please note that there are temporary firewall rules within this configuration that should be removed once the entire Bank of Anthos deployment is complete.**
```

Comment:
We should clearly state this is not a full config and expected to be used in conjunction with example foundation networking layer in hub and spoke configuration